package.json

```diff
@@ -41,6 +41,6 @@
     "file-loader": "^0.8.5",
     "style-loader": "^0.13.1",
     "url-loader": "^0.5.7",
-    "webpack": "^1.13.1"
+    "webpack": "^1.15.0"
```

Comment:
Is this required?

Reply:
I dont know for sure,I didn't specified the version while installing webpack.
Can you  review the PR, i have removed the unnecessary changes.